subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -171,6 +171,11 @@ uint8_t ll_adv_aux_ad_data_set(uint8_t handle, uint8_t op, uint8_t frag_pref, ui
 	if (!is_aux_new) {
 		sec_hdr_prev = *sec_hdr;
 	} else {
+		sec_pdu_prev->chan_sel = 0U;
```

Comment:
Not setting `rfu = 0`

Reply:
Updated code with comment, now initializing only fields used to copy into new PDU buffer.